ios/VydiaRNFileUploader.m

```diff
@@ -137,6 +135,37 @@ - (NSString *)guessMIMETypeFromFileName: (NSString *)fileName {
             }
         }];
 
+        // asset library files have to be copied over to a temp file.  they cannot be uploaded directly
+        if ([fileURI hasPrefix:@"assets-library"]) {
+            NSURL *url = [NSURL URLWithString:fileURI];
+            PHAsset *asset = [PHAsset fetchAssetsWithALAssetURLs:@[url] options:nil].lastObject;
+            if (!asset) {
+                reject(@"RN Uploader", @"Asset could not be fetched.  Are you missing permissions?", nil);
+                return;
+            }
+            PHAssetResource *assetResource = [[PHAssetResource assetResourcesForAsset:asset] firstObject];
+            NSString *pathToWrite = [NSTemporaryDirectory() stringByAppendingPathComponent:[[NSUUID UUID] UUIDString]];
+            NSURL *pathUrl = [NSURL fileURLWithPath:pathToWrite];
+            fileURI = pathUrl.absoluteString;
```

Comment:
What is the format or value of `fileURI` after being set to `pathUrl.absoluteString;`?

Reply:
file://blahblahblah